gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -286,15 +286,23 @@ def add_arguments(self, parser):
         type='bool', default=False, nargs='?', const=True,
         help=('If true, runs annotation tools (currently only VEP) on input '
               'VCFs before loading to BigQuery.'))
+    parser.add_argument(
+        '--shard_input_files',
+        type='bool', default=True, nargs='?', const=True,
+        help=('If true, shard the input files before running annotation tools. '
+              'This can improve the annotation performance for large input '
+              'files.'))
     parser.add_argument(
         '--' + AnnotationOptions._OUTPUT_DIR_FLAG,
         default='',
         help=('The path on Google Cloud Storage to store annotated outputs. '
               'The output files are VCF and follow the same directory '
-              'structure as input files with a suffix added to them.'))
+              'structure as input files with a suffix added to them. Note that '
+              'this is expected not to exist and is created in the process of '
+              'running VEP pipelines.'))
     parser.add_argument(
         '--' + AnnotationOptions._VEP_IMAGE_FLAG,
-        default='gcr.io/gcp-variant-annotation/vep_91',
+        default='gcr.io/gcp-variant-annotation/vep_91_in_vt',
```

Comment:
I would remove _in_vt suffix. The fact that it has watchdog logic should not make it specific to VT.